tests/utils.spec.ts

```diff
@@ -248,11 +248,8 @@ describe('utils tests', () => {
   })
 
   it('test: getTsLibFolder', () => {
-    const root = normalizePath(resolve(__dirname, '..'))
-    const entryRoot = resolve(root, 'src')
-
-    expect(getTsLibFolder({ root, entryRoot })).toMatch(/node_modules\/typescript$/)
+    expect(getTsLibFolder()).toMatch(/node_modules\/typescript$/)
 
-    expect(existsSync(getTsLibFolder({ root, entryRoot }) || '')).toBe(true)
+    expect(existsSync(getTsLibFolder() || '')).toBe(true)
   })
```

Comment:
Not related to the main change, but `tsc` was complaining because `getTsLibFolder` no longer accepts parameters as of 0621332